src/lumigo_tracer/lambda_tracer/tracer.py

```diff
@@ -37,7 +36,6 @@ def _add_wrap_flag_to_context(*args):  # type: ignore[no-untyped-def]
 def _lumigo_tracer(func):  # type: ignore[no-untyped-def]
     if is_kill_switch_on():
         return func
-    wrap()
```

Comment:
This code was running in global scope (when someone wrap its lambda with `lumigo_tracer()`.
Moved to the global_exec